python/pyspark/testing/utils.py

```diff
@@ -396,6 +398,12 @@ def assertDataFrameEqual(
     checkRowOrder: bool = False,
     rtol: float = 1e-5,
     atol: float = 1e-8,
+    ignoreNullable: bool = True,
+    ignoreColumnOrder: bool = False,
+    ignoreColumnName: bool = False,
+    ignoreColumnType: bool = False,
```

Comment:
have we considered `ignoreSchema`?

Reply:
It sounds worth to have it for some scenarios. Let's discuss in the separate thread!